app/components/primer/alpha/select_panel.html.erb

```diff
@@ -13,7 +13,7 @@
             <%= render(Primer::BaseComponent.new(
               tag: :"remote-input",
               aria: { owns: @body_id },
-              **(@fetch_strategy == :remote ? { src: @src } : {}),
+              **(@fetch_strategy == :local ? {} : { src: @src }),
```

Comment:
For `eventually_local` panels, loading is done via an `<include-fragment>`, which is why `src` isn't attached to the `<remote-input>` element in that case.

Reply:
Ah, helpful -- I think the problem in that case is the spinner hiding function call only fires when the input itself gets the `loadend` event. Ended up just explicitly saying "hide the spinner" after the `include-fragment` is done, which I think is easier to follow than making `handleEvent` even bigger.

Reply:
Ah ok that makes sense 👍